buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1914,6 +1914,10 @@ class BeamModulePlugin implements Plugin<Project> {
     // Method to create the crossLanguageValidatesRunnerTask.
     // The method takes crossLanguageValidatesRunnerConfiguration as parameter.
     project.ext.createCrossLanguageValidatesRunnerTask = {
+      // This task won't work if the python build file doesn't exist.
+      if (!project.project(":sdks:python").buildFile.exists()) {
+        return
```

Comment:
I thought maybe we should return a task that errors out when you try to call it, instead of leaving the task mysteriously missing. But it's probably not worth the effort.

Reply:
Yeah, the complexity added to the code may not be worth it. I appended the log message for providing some hints.